utils/single_module.py

```diff
@@ -215,7 +215,7 @@ def test_mode_module(name, interactive):
     else:
         try:
             fame_init()
-            module = dispatcher.get_processing_module(args.module)
+            module = dispatcher.get_module(args.module)
```

Comment:
All changes inside this file should be reverted since this PR is not providing support for preloading modules in `single_module.py`

Reply:
file reverted